packages/core/util/tests/util-specs.js

```diff
@@ -68,6 +122,46 @@ describe('util', () => {
 		});
 	});
 
+	describe('mergeClassNameMaps', () => {
+		const baseMap = {
+			'class-base-only': 'real-class-base-only',
+			'class-shared': 'real-class-shared-base',
+			'class-shared-another': 'real-class-shared-another-base'
+		};
+		const additiveMap = {
+			'class-shared': 'real-class-shared-additive',
+			'class-shared-another': 'real-class-shared-another-additive',
+			'class-additive-only': 'real-class-additive-only'
+		};
+
+		test('should return a base map if an additive map is not given', () => {
+			const expected = mergeClassNameMaps(baseMap);
+
+			expect(expected['class-base-only']).toEqual('real-class-base-only');
+			expect(expected['class-shared']).toEqual('real-class-shared-base');
+			expect(expected['class-shared-another']).toEqual('real-class-shared-another-base');
+			expect(expected['class-additive-only']).not.toEqual('real-class-additive-only');
+		});
+
+		test('should return a merged map containing shared class names', () => {
+			const expected = mergeClassNameMaps(baseMap, additiveMap);
```

Comment:
Also, shouldn't it be 'actual'?

Reply:
Can we make an expected object to compare? Please refer to `usePublicClassNames-specs`.

Reply:
Ah, not I got your point. Good idea.

Reply:
I created such a long test code since the below code blocks to change the result.
https://github.com/enactjs/enact/blob/develop/packages/core/util/util.js#L195
Rather than the merged `css` object, it returns a Proxy instance, so the returned value is actually an empty object and but we can access its properties via getters.

Reply:
I added a way to avoid proxy instance in unit test code, so finally it compares via `toEqual` between objects.